### PR TITLE
feat: Explain colors in diff view DOCS-547

### LINKED ISSUE #1682: [DOCS-547] Explain how to read the colors on the Diff tab
**Type of feedback**

-   [ ] Bug or incorrect information
-   [X] Missing information
-   [ ] Outdated information
-   [ ] Enhancement
-   [ ] Other

**Feedback**

We should explain the meaning of the colors that we use on the Diff tab (of commits and pull requests).

**Affected pages**
This feedback applies to the following documentation pages / URLs:

- https://docs.codacy.com/repositories/commits/#diff-tab
- https://docs.codacy.com/repositories/pull-requests/#diff-tab

**Additional context**
If applicable, provide any other context or information that could be useful in updating the documentation:

This feedback was originally submitted on 2023-04-17 by a user on the page https://docs.codacy.com/repositories/pull-requests/#diff-tab

### PR DESCRIPTION
**Pull request by prcr**
Fixes #1682.

### :eyes: Live preview
- https://feat-explain-diff-colors-docs-547--docs-codacy.netlify.app/repositories/commits/#diff-tab
- https://feat-explain-diff-colors-docs-547--docs-codacy.netlify.app/repositories/pull-requests/#diff-tab

### CODE REVIEW

File: docs/repositories/commits.md [hunk 1]
```diff
@@ -125,7 +125,12 @@ The option **Show files without code quality changes** allows you to list all fi
 
 ## Diff tab
 
-The **Diff** tab displays the line differences in each file that was changed in the {{ page.meta.page_name }}.
+The **Diff** tab displays the differences in each file that was changed in the {{ page.meta.page_name }}. The background of the lines that are different depend on the change:
```
Comment (nicklem) [inline docs/repositories/commits.md:128]: Tweak for clarity:

```suggestion
The **Diff** tab displays the differences in each file that was changed in the {{ page.meta.page_name }}. The background of the changed lines depends on the change:
```

File: docs/repositories/commits.md [hunk 2]
```diff
@@ -125,7 +125,12 @@ The option **Show files without code quality changes** allows you to list all fi
...
+
+-   **Red**: Deleted line
+-   **Yellow**: Original version of a changed line with the updated characters highlighted in red
+-   **Pale green**: New version of a changed line with the updated characters highlighted in bright green
```
Comment (nicklem) [inline docs/repositories/commits.md:132]: If this behaves anything like the GitHub version (to me looks like it does after double-checking), I'd tweak as follows for clarity:

```suggestion
-   **Yellow**: Old version of a changed line with deleted characters highlighted in red
-   **Pale green**: New version of a changed line with added characters highlighted in bright green
```

File: docs/repositories/commits.md (hunk 1 above)
Comment (prcr) [reply docs/repositories/commits.md:128]: Indeed, much better!

File: docs/repositories/commits.md [hunk 3]
```diff
@@ -125,7 +125,12 @@ The option **Show files without code quality changes** allows you to list all fi
 
 ## Diff tab
 
-The **Diff** tab displays the line differences in each file that was changed in the {{ page.meta.page_name }}.
+The **Diff** tab displays the differences in each file that was changed in the {{ page.meta.page_name }}. The background of the changed lines depends on the change:
+
+-   **Red**: Deleted line
+-   **Yellow**: Old version of a changed line with deleted characters highlighted in red
+-   **Pale green**: New version of a changed line with added characters highlighted in bright green
```
Comment (prcr) [inline docs/repositories/commits.md:132]: I'm thinking that maybe it's not worth distinguishing between "pale green/bright green" and we can have just "green/bright green" instead. What do you think?

```suggestion
-   **Green**: New version of a changed line with added characters highlighted in bright green
```

**Review by nicklem — APPROVED** [review]
LGTM :shipit:

**Pull request merged**